goleft_functions.wdl

```diff
@@ -53,6 +53,12 @@ task indexcovCRAM {
 	Int thisAmSize = ceil(size(inputCram, "GB"))
 	Int finalDiskSize = indexSize + thisAmSize + indexcovAddlDisk
 
+	# Basename including extension but exclude preceeding folders
+	String cramBasename = basename(inputCram)
+
+	# Prefix of output files and directory, ie, cramBasename minus extension
+	String prefix = basename(sub(inputCram, "\.cram(?!.{5,})", ""))
```

Comment:
This might be very very long.  Suggestion: if you just want a unique name, the date/time would have a defined and short-ish length.

Reply:
That would be shorter, but it also would prevent the checker workflow template from working, as it matches test with truth files under the assumption they share a name.